src/styles/bookItem.css

```diff
@@ -0,0 +1,104 @@
+.book {
+  border: rgb(160, 158, 158) 1px solid;
+  background-color: white;
+  margin-bottom: 19px;
+  width: 90%;
+  padding-left: 2%;
+  padding-right: 2%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.titleP {
+  color: black;
+  font-size: large;
+  margin-top: -12px;
+  margin-bottom: 22px;
+}
+
+.categoryP {
+  color: rgb(141, 140, 140);
+}
+
+.bookButton {
+  border: none;
+  background-color: inherit;
+  cursor: pointer;
+  color: rgb(64, 152, 224);
+}
```

Comment:
- [ ] A margin on the bottom could help to make your style more according to the template, please add it. 🌵 
![image](https://user-images.githubusercontent.com/20563252/135684146-8fd5db71-462a-4db9-84b5-200af3ce10f1.png)

- [ ] At the same time, try to add the lines between each element on this part. This will help to make a division on the options. 🌴 
![image](https://user-images.githubusercontent.com/20563252/135684309-ff85a36f-d3a0-4999-acef-1d7a9e4841b2.png)

- [ ] The author is not visible, please add it. 🥥 
![image](https://user-images.githubusercontent.com/20563252/135684400-0622e646-378c-439d-a9fe-72c2c6bd99c7.png)

Reply:
while we have a provision for the author on the mockup, we do not have it in the back end that's why I didn't add it